client/src/js/templates/shipment/transfer.html

```diff
@@ -81,7 +81,7 @@ <h1>Request Internal Dewar Transfer</h1>
             <li class="head">Next Visit Details</li>
             <li>
                 <label>Next Visit
-                    <span class="small">The next visit the dewar is going to be used for</span>
+                    <span class="small">Select "-" to return the dewar to the responsive scheduling queue</span>
```

Comment:
Word this as "leave blank" instead of "select -" maybe? Or we could change the default select option to "return to responsive scheduling queue", if that doesn't break anything else elsewhere

Reply:
I like "Leave Blank", it looks weird with the quotes around the dash. I thought changing the option text would look a little odd, and there might be other reasons to leave it blank as well.